apps/src/templates/projects/projectsRedux.js

```diff
@@ -463,6 +469,27 @@ export function unpublishProject(projectId) {
   };
 }
 
+export const unpublishProjectLibrary = (
+  projectId,
+  libraryApi = new LibraryClientApi(projectId),
+  onComplete = () => {}
+) => {
+  return dispatch => {
+    fetchProjectToUpdate(projectId, (error, data) => {
+      if (error) {
+        onComplete(error, data);
+      } else {
+        libraryApi.unpublish(data, (error, serverData) => {
```

Comment:
I think you can just call unpublish. There _should_ be different error codes if the project just doesn't exist (404) vs if there is a server error.

Reply:
i'm not sure what you mean -- can you rephrase that?

Reply:
Nevermind. I had the wrong thing in mind for what fetchProjectToUpdate does.